tests/src/test/scala/com/evolutiongaming/kafka/journal/IntegrationSuite.scala

```diff
@@ -14,52 +12,50 @@ import com.evolutiongaming.scassandra.CassandraClusterOf
 import com.evolutiongaming.smetrics.CollectorRegistry
 import com.github.dockerjava.api.model.{ExposedPort, HostConfig, PortBinding, Ports}
 import com.typesafe.config.ConfigFactory
+import org.testcontainers.cassandra.CassandraContainer
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.kafka.KafkaContainer
+import org.testcontainers.utility.DockerImageName
 
 object IntegrationSuite {
+  private val CassandraDockerImage = DockerImageName.parse("cassandra:4.1.8") // latest 4.x
+  private val KafkaDockerImage     = DockerImageName.parse("apache/kafka-native:3.8.1") // 3.9.0 doesn't start here
```

Comment:
what about 4.0.0? :)

Reply:
I didn't try, not sure I want to. We don't use 4.x Kafka anywhere, I don't think it makes sense to test with it.

Reply:
what's the problem with 3.9.0?